### DISCUSSION TITLE
[Question][Hw3] Unbound value when calling debug_location

### DISCUSSION THREAD

**Discussion by mrpositron** [prosyslab-classroom/cs348-information-security#532] [Q&A]
When I define `debug_location` like this in `instrument.ml`.
```
let dbg_info =  debug_location llctx instr in ...
```
It returns following:
```
Error: Unbound value utils
```
Could you help me?

**Comment by Junseon00** [comment]
You can use`Utils.debug_locations` or just import Utils on your `instrument.ml`

**Comment by mrpositron** [comment]
Thanks! It works.
